fb-negotiation/neural/model_builder.py

```diff
@@ -0,0 +1,342 @@
+"""
+This file is for models creation, which consults options
+and creates each encoder and decoder accordingly.
+"""
+import torch
+import torch.nn as nn
+import pdb
+import onmt
+import onmt.io
+import onmt.Models
+import onmt.modules
+from onmt.modules import Embeddings, ImageEncoder, CopyGenerator
+
+from cocoa.neural.models import MeanEncoder, StdRNNEncoder, StdRNNDecoder, \
+              MultiAttnDecoder, NMTModel
+from models import FBNegotiationModel, LM
+
+from cocoa.io.utils import read_pickle
+from cocoa.pt_model.util import use_gpu
+
+from symbols import markers
+from neural import make_model_mappings
+
+def add_model_arguments(parser):
+    from onmt.modules.SRU import CheckSRU
```

Comment:
import existing arguments from cocoa.neural.model_builder. only add those specific to fb here.